### DISCUSSION TITLE
Rules to auto-route to browser via regex

### DISCUSSION THREAD

**Discussion by lzilioli** [will-stone/browserosaurus#485] [Ideas]
Hi, really enjoying the app so far. Thank you for creating it.

Certain links, such as those for my company's website, I ALWAYS want to open in a particular browser. At present, there is no way to tell browsersarus "all links that contain XXX should open in Chrome". This would be a great addition to the app.

**Comment by will-stone** [comment]
Hi. Thanks for using Browserosaurus. Please see here: https://github.com/will-stone/browserosaurus/discussions/476
